core/src/test/scala/org/broadinstitute/dsde/rawls/mock/MockSamDAO.scala

```diff
@@ -171,6 +171,10 @@ class MockSamDAO(dataSource: SlickDataSource)(implicit executionContext: Executi
 
   override def getUserArbitraryPetServiceAccountKey(userEmail: String): Future[String] = ???
 
+  override def getUserPetServiceAccount(ctx: RawlsRequestContext,
+                                        googleProjectId: GoogleProjectId
+  ): Future[WorkbenchEmail] =
+    Future.successful(WorkbenchEmail("pet-110347448408766049948@broad-dsde-dev.iam.gserviceaccount.com"))
```

Comment:
Is the specific email address meaningful in any way?

Reply:
it was copied from other tests in rawls..... 🤷

Reply:
OK, I just wondered because whenever I see something that isn't obviously fake I wonder if it means that there was some setup someplace with this specific account (so the test is dependent on that). So if the test passes with an arbitrary email address I would suggest changing it, but it's not a big deal.